src/views/Users.vue

```diff
@@ -16,24 +16,24 @@
         </b-row>
         <b-row class="name-wrapper">
           <label for="name">Name</label>
-          <b-input></b-input>
+          <b-input v-model="nameFilter"></b-input>
         </b-row>
         <b-row class="email-wrapper">
           <label for="email">Email</label>
```

Comment:
for="" attribute is not linking the label with the field since the input does not have an id

Reply:
Done